staging/src/k8s.io/apiserver/pkg/storage/etcd3/watcher_test.go

```diff
@@ -382,14 +387,41 @@ func testCheckEventType(t *testing.T, expectEventType watch.EventType, w watch.I
 	}
 }
 
+func resourceVersionNoLaterThan(sentinel string) func(string) error {
```

Comment:
am I misreading this, or is this actually returning a function that enforces "not earlier than $sentinel"

Reply:
This is imprecise language. I will update it to use older/earlier

Reply:
should this be named `resourceVersionNoOlderThan` instead?